### PR TITLE
Mobs start with a random nutrition amount

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
balance: Mobs will now start with a random nutrition amount, between
hungry and mildly well fed.
/:cl:

Basically, if everyone starts the round with 400 nutrition, then
everyone will likely get hungry at around the same time if they're
standing around. This means there's a little more variance in our
starting values, which means the kitchen could be more important a
little bit sooner.

### CODE REVIEW

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -72,10 +72,14 @@
 #define NUTRITION_LEVEL_FAT 600
 #define NUTRITION_LEVEL_FULL 550
 #define NUTRITION_LEVEL_WELL_FED 450
+#define NUTRITION_LEVEL_MAXIMUM_STARTING 400
```
Comment (YPOQ) [inline code/__DEFINES/mobs.dm]: Duplicate?

File: code/modules/mob/mob_defines.dm [hunk 2]
```diff
@@ -58,7 +58,7 @@
 	var/drowsyness = 0//Carbon
 	var/dizziness = 0//Carbon
 	var/jitteriness = 0//Carbon
-	var/nutrition = NUTRITION_LEVEL_FED + 50//Carbon
+	var/nutrition = rand(NUTRITION_LEVEL_START_MIND, NUTRITION_LEVEL_START_MAX)
```
Comment (YPOQ) [inline code/modules/mob/mob_defines.dm]: MIND->MIN

File: code/modules/mob/mob_defines.dm [hunk 3]
```diff
@@ -58,7 +58,7 @@
 	var/drowsyness = 0//Carbon
 	var/dizziness = 0//Carbon
 	var/jitteriness = 0//Carbon
-	var/nutrition = NUTRITION_LEVEL_FED + 50//Carbon
+	var/nutrition = rand(NUTRITION_LEVEL_START_MIN, NUTRITION_LEVEL_START_MAX)
```
Comment (DaxDupont) [inline code/modules/mob/mob_defines.dm]: This shouldn't be in defines, it requires constants. not variable things.

Comment (coiax) [reply code/modules/mob/mob_defines.dm]: The definition aspect is in the creation of the variable, not its value.

**Comment by KomradeSpectre** [comment]
Kind of strange seeing a PR by you that I agree with, Coiax, but this is nice as it will give the chef a bit more importance throughout the round.

**Comment by PKPenguin321** [comment]
By "hungry" do you mean already slowed from hunger or in the orange range?

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/mob/mob_defines.dm (hunk 3 above)
Comment (Cyberboss) [inline code/modules/mob/mob_defines.dm]: You have to do this in Initialize actually

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Jalleo** [comment]
Could we have it overriden for the ghost roles and off station antags?

**Comment by Nabski89** [comment]
This has fucked up xenobiology so that now slimes require three to split.

**Comment by Bawhoppen** [comment]
So?

**Pull request merged**